static/js/src/openstack/react/components/CostCalculations.jsx

```diff
@@ -0,0 +1,293 @@
+import React from "react";
+import { Row, Col } from "@canonical/react-components";
+
+import TCO_VARIABLES from "../utils/variables";
+
+const CostCalculations = (formState) => {
+  //RAM
+  const amountOfRamInCloud = formState.instances * formState.ram;
+
+  const amountOfRamInFullyUtilizedCloud = Math.ceil(
```

Comment:
You use Math.ceil for each intermediate value. Is it defined this way in the spreadsheet? Otherwise you could only use Math.ceil on the final computed value to minimise rounding errors

Reply:
It is defined like this on the spreadsheet, but I think it makes sense to leave it on the final computed value like you say will reduce rounding errors